jdaviz/configs/default/plugins/export/export.py

```diff
@@ -187,6 +190,22 @@ def _sync_singleselect(self, event):
                 setattr(self, attr, '')
             if attr == 'subset_selected':
                 self._set_subset_not_supported_msg()
+            elif self.config == "cubeviz" and attr == "viewer_selected":
+                if self.viewer_selected == "spectrum-viewer" and self.viewer_format_selected == "png":
+                    msg = "Exporting the spectrum viewer as a PNG in Cubeviz is currently disabled"
+                else:
+                    msg = ""
+                self.viewer_invalid_msg = msg

```

Comment:
Can we just call `_disable_viewer_format_combo` to be consistent with the other cases?

Reply:
Done, good thought.